metro.transform.js

```diff
@@ -10,6 +10,7 @@ module.exports.transform = function applyRainbowTransform({
   const opts = merge(options, {
     customTransformOptions: {
       'metro-plugin-anisotropic-transform': {
+        cyclicDependents: /.+\/node_modules\/react-native\/Libraries\/BatchedBridge\/NativeModules\.js$/,
```

Comment:
Just a heads up: we're changing this to run on CI only so it runs faster locally via #1771